### PR TITLE
fix: #1133:  All the carts should be of same height

### PR DESCRIPTION
**Pull request by subhamkumarr**
## Description:
All the carts is having same height

#### Before:
Screenshot (1098)

#### After:
image

### CODE REVIEW

**Comment by subhamkumarr** [comment]
@alexandratran could you please review the PR

File: src/components/Card.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export type CardItem = {
 export default function Card({ title, link, description }: CardItem) {
   return (
     <div className="col col--4 margin-top--sm margin-bottom--md">
-      <Link className={clsx(styles.root, "card padding--sm")} href={link}>
+      <Link className={clsx(styles.root, "card")} href={link}>
```
Comment (alexandratran) [inline src/components/Card.tsx:15]: @subhamkumarr It looks like the homepage cards have small padding by default, so `padding--sm` is redundant and I can merge this change if you want. However, this doesn't quite fix the issue of cards being different heights – the card height still depends on the amount of text in the card + the width of your screen.

Comment (subhamkumarr) [reply src/components/Card.tsx:15]: By fixing this even i observe that the card height depends on the amount of text in the card. I look into all other card this change does not harm other card. 

I want this to be merge. This change making home page look beautiful.

**Review by alexandratran — APPROVED** [review]

**Comment by alexandratran** [comment]
@subhamkumarr Sorry, I do not help maintain the `metamask-extension` repo. Just give them a little more time or ping on the [Consensys Discord](https://discord.gg/consensys).

**Pull request merged**